pkg/runner/cluster_k8s.go

```diff
@@ -700,7 +700,7 @@ func (c *ClusterK8sRunner) monitorTestplanRunState(ctx context.Context, ow *rpc.
 		default:
 		}
 
-		if time.Since(start) > 10*time.Minute {
+		if time.Since(start) > 100*time.Minute {
```

Comment:
@raulk @nonsense this was a change I had made since it seemed useful at the time. Worth keeping it like this until we have a config option for it?

Reply:
I will add a config in `.env.toml` for it.

Reply:
we want this too! We want to run tests for longer than 10m.